theme/templates/accounts/sign_up_form.html

```diff
@@ -47,13 +67,31 @@
         });
 
         $("#signup").click(function() {
-            var username = $("#username").val();
-            var firstName = $("#firstname").val();
-            var lastName = $("#lastname").val();
-            var email = $("#email").val();
-            var password = $("#password").val();
-            var pwconfirm = $("#pwconfirm").val();
-
+            var password = $("#id_password1").val();
+            var pwconfirm = $("#id_password2").val();
+            var fieldsToCheck = ["#id_username", "#id_email", "#id_password1", "#id_password2",
+                "#recaptcha_response_field"];
+            var fieldNames = [ 'User name', 'Email', 'Password', 'Password (again)', 'ReCaptcha'];
+            var empty = false;
+            var missingFields = 'Data missing for:';
+            var missingFieldCount = 0;
+            var j;
+            for(j=0; j< fieldsToCheck.length; j++){
+                if ($(fieldsToCheck[j]).val() == ''){
+                    empty = true;
+                    if (missingFieldCount == 0){
+                        missingFields += " " + fieldNames[j];
+                    }
+                    else {
+                        missingFields += ", " + fieldNames[j];
+                    }
+                    missingFieldCount++;
+                }
+           }
+           if(empty){
+               $("#missing-data-div").html('<p class="alert alert-danger">' + missingFields +'</p>');
+               return;
+           }
```

Comment:
@pkdash I can see some styling issue, like no space before ```{```. Looks like this file is not subject to pylint check. Try to remove it from the config file so that the styles can be checked by pylint.